ed_sensor_integration/src/kinect/fitter.cpp

```diff
@@ -248,6 +234,7 @@ Fitter::Fitter(uint nr_data_points) :
     nr_data_points_(nr_data_points)
 {
     beam_model_.initialize(4, nr_data_points);
+    configured_ = false;
 }
```

Comment:
I think this constructor isn't valid anymore. As you want to configure it with the camera model.